.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -38,3 +38,4 @@ implementation proposal in the referenced issues.
 - [ ] Documentation: Relevant documentation is added or updated.
 - [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
 - [ ] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.
+- [ ] Target: If the changes are not changing any API, configuration and storage, the target branch is set to master, otherwise the target branch is set to develop.
```

Comment:
I think there's a bit too much overlap with the "Compatibility" checkbox. Maybe we should combine the two?

Reply:
Yeah good one. I've combined them